Remotion/Web/Core/Res/Themes/NovaViso/HTML/Common.css

```diff
@@ -0,0 +1,109 @@
+:root {
+    /*colors*/
+    --primary: #0C6594;
+    --highlight: #DEF3FE;
+    --window-border: #C4C4C4;
+    --separator: #E1E1E1;
+    --menu-background: #F9F9F9;
+    --hover: #218FC9;
+    --border: #979797;
+    --dossier-section-yellow: #DFAF00;
+    --dossier-section-red: #DA2222;
+    --dossier-section-green: #4AD439;
+    --focus: #23ADF8;
+    --header: #F2F2F2;
+
+    --dropdown-scrollbar-track: #E3E3E3;
+    --scrollbar-track: #D3D3D3D3;
+    --background: #FFFFFF;
+
+    /*remotion-themed*/
+    --remotion-themed-border-radius: 2px;
+    --remotion-themed-border-width: 1px;
+    --remotion-themed-border: var(--remotion-themed-border-width) solid var(--border);
+    --remotion-themed-border-focus: var(--remotion-themed-border-width) solid var(--focus);
+    --remotion-themed-border-hover: var(--remotion-themed-border-width) solid var(--primary);
+    --remotion-themed-dropdown-scrollbar-border-radius: 10px;
+    --remotion-themed-dropdown-scrollbar-width: 10px;
+    --remotion-themed-scrollbar-width: 14px;
+}
+
+input.remotion-themed,
+textarea.remotion-themed,
+select.remotion-themed,
+.remotion-themed input,
+.remotion-themed textarea,
+.remotion-themed select {
+    border: var(--remotion-themed-border);
+    border-radius: var(--remotion-themed-border-radius);
+}
+
+input.remotion-themed:hover,
+textarea.remotion-themed:hover,
+select.remotion-themed:hover,
+.remotion-themed input:hover,
+.remotion-themed textarea:hover,
+.remotion-themed select:hover {
+    border: var(--remotion-themed-border-hover);
+    outline-style: none;
+}
+
+input.remotion-themed:focus,
+textarea.remotion-themed:focus,
+select.remotion-themed:focus,
+.remotion-themed input:focus,
+.remotion-themed textarea:focus,
+.remotion-themed select:focus {
+    border: var(--remotion-themed-border-focus);
+    outline-style: none;
```

Comment:
The outline disable doesn't work in Firefox for radio & checkbox
The focus highlight doesn't work in Edge/Chrome for radio & checkbox

If we can't get the border color to work, we still skip the outline for now and add this to the follow-up task. I'm pretty sure we will want to have an outline when we're done, but that's out of scope for the current task.